playgrounds/sandbox/scripts/create-test.js

```diff
@@ -0,0 +1,119 @@
+// Creates a test from the existing playground. cwd needs to be playground/sandbox
+import fs from 'fs';
+import path from 'path';
+
+// Get target folder from command line arguments
+let target_folder = process.argv[2];
+if (!target_folder) {
+	console.error(
+		'Please provide a target folder as an argument. Example: node create-test.js runtime-runes/my-test'
+	);
+	process.exit(1);
+}
+if (!target_folder.includes('/')) {
+	target_folder = 'runtime-runes/' + target_folder;
+}
+if (!target_folder.startsWith('runtime-')) {
+	console.error(
+		'Target folder must start with "runtime-" (can only convert to these kinds of tests)'
+	);
+	process.exit(1);
+}
+target_folder = path.join(
+	path.resolve('../../packages/svelte/tests', target_folder.split('/')[0]),
+	'samples',
+	target_folder.split('/')[1]
+);
+
+// Create target directory if it doesn't exist
+if (!fs.existsSync(target_folder)) {
+	fs.mkdirSync(target_folder, { recursive: true });
+}
```

Comment:
Wouldn't it be better to error out if the target folder already exists to avoid overwriting an existing test?

Reply:
added logic to ask if we should override (defaults to yes because this usually happens when you wanna update something)